src/main/scala/org/codeoverflow/chatoverflow/ChatOverflow.scala

```diff
@@ -61,7 +61,10 @@ class ChatOverflow(val pluginFolderPath: String,
   }
 
   private def askForPassword(): Unit = {
-    val password = scala.io.StdIn.readLine("Please enter password > ").toCharArray
+    val password = if (System.console() != null)
+        System.console().readPassword("Please enter password (Input hidden) > ")
```

Comment:
Hi there! 
I think there is a .toCharArray missing ^^

Reply:
`readPassword` already returns a char array. See [here](https://docs.oracle.com/javase/9/docs/api/java/io/Console.html#readPassword-java.lang.String-java.lang.Object...-)

Reply:
Oh, my bad.

Reply:
No problem